src/domain/runtime/type/remote_object/mod.rs

```diff
@@ -88,7 +88,7 @@ impl fmt::Display for RemoteObject {
                 if let Some(value) = &self.value {
                     let value = value.to_string();
                     let end = value.len() - 1;
-                    value[1..end].to_string()
+                    value[1..end].to_string().replace("\\", "")
```

Comment:
what does this to do `\n`? does it just change to `\`?

Reply:
So this would only effect `\` characters if they're in the string. So `"Test\n"` would not be affected unless there was a backslash and n character in the string instead of a newline character.

Reply:
Yeah that would probably be a better idea, although in the case where they intend to have the characters \ and n be printed out then it would print a newline instead. So maybe we should only do it for \"?

Reply:
mm - i think we should just honor every escape sequence there is. that's what node does.

```console
$ node
> console.log("\"Hello,\n\tWorld!\"")
"Hello,
        World!"
```

Reply:
probably want to use [this](https://docs.rs/snailquote/0.3.0/snailquote/fn.unescape.html)

Reply:
So my change wouldn't actually effect the string you gave above. What's most likely happening is whoever is sending the request is double json serializing resulting in \\\" becoming actual characters in the string. Not just the representation of ". So it would be more like if you console logged the following. 
```javascript
Welcome to Node.js v14.3.0.
Type ".help" for more information.
> test = [{"wee": "dflksjd"}, {"woo": "weifojwifej"}]
[ { wee: 'dflksjd' }, { woo: 'weifojwifej' } ]
> json = JSON.stringify(test)
'[{"wee":"dflksjd"},{"woo":"weifojwifej"}]'
> JSON.stringify(json)
'"[{\\"wee\\":\\"dflksjd\\"},{\\"woo\\":\\"weifojwifej\\"}]"'
```
The reason I'm adding replace is so that I can get ride of \\ that are in the string so it can be easily deserialized.

Reply:
In the case you gave above it would be like
```
> console.log("\\\"Hello,\\n\\tWorld!\\\"")
```